frontend/src/entry/overview/EntryCardEntityRoleRow.tsx

```diff
@@ -0,0 +1,45 @@
+import GroupIcon from '@mui/icons-material/Groups'
+import UserIcon from '@mui/icons-material/Person'
+import { Grid, Stack } from '@mui/material'
+import { Fragment, useMemo } from 'react'
+import UserDisplay from 'src/common/UserDisplay'
+import EntryCardRolesChipSet from 'src/entry/overview/EntryCardRolesChipSet'
+import { CollaboratorEntry } from 'types/types'
+
+type EntryCardEntityRoleRowProps = {
+  entryCollaborators: CollaboratorEntry[]
+}
+
+export default function EntryCardEntityRoleRow({ entryCollaborators }: EntryCardEntityRoleRowProps) {
+  return entryCollaborators.map((user) => (
+    <Fragment key={user.entity}>
+      <Grid item xs={6}>
+        <Stack direction='row' alignItems='center' spacing={1}>
+          <EntryCollaboratorIcon entity={user} />
+          <EntryCollaboratorName entity={user} />
+        </Stack>
+      </Grid>
+      <Grid item xs={6}>
+        <EntryCardRolesChipSet entryCollaborator={user} />
+      </Grid>
+    </Fragment>
+  ))
+}
+
+type EntryCollaboratorProps = {
+  entity: CollaboratorEntry
+}
+
+function EntryCollaboratorIcon({ entity }: EntryCollaboratorProps) {
```

Comment:
Instead of duplicating code by adding `EntryCollaboratorIcon` and `EntryCollaboratorName` in here you should instead move `EntityIcon` and `EntityNameDisplay` from `frontend/src/entry/settings/EntityItem.tsx` into new files in `frontend/src/entry/EntityIcon.tsx` and `frontend/src/entry/EntityIcon.tsx` respectively. I would then export them from there and use them here and in `EntityItem.tsx`.

While you're doing that I also suggest fixing the props. They're collaborator entries, not entities. So consider renaming them from `entity: CollaboratorEntry` to `entryCollaborator: CollaboratorEntry`.